UserService/UserService.UnitTests/GetHelpersByPostcodeHandlerTests.cs

```diff
@@ -82,7 +84,7 @@ public void SetUp()
             _addressService = new Mock<IAddressService>();
 
             _addressService.Setup(x => x.GetPostcodeCoordinatesAsync(It.IsAny<GetPostcodeCoordinatesRequest>(), It.IsAny<CancellationToken>())).ReturnsAsync(_getPostcodeCoordinatesResponse);
-
+            _helperService = new HelperService(_addressService.Object, _volunteerCache.Object, _distanceCalculator.Object, _repository.Object);
```

Comment:
As related to my other comment, you have to manually set up IHelperService here, which is extra effort when it would be much easier just to mock it.  It is fine to mock if there are separate tests for the HelperService class.  Again, when the HelperService is used again in a third handler it is extra effort to manually set it up for a test.

Reply:
implemented seperate tests for helper service now